packages/devtools/lib/src/timeline/timeline.css

```diff
@@ -89,6 +89,37 @@
     cursor: default;
 }
 
+.flame-chart-grid {
+    left: 0;
+    top: 0;
+    position: absolute;
+    min-width: 100%;
+    min-height: 100%;
+    pointer-events: none;
+}
+
+.flame-chart-grid-item {
+    top: 0;
+    min-height: 100%;
+    position: absolute;
+    pointer-events: none;
+}
+
+.flame-chart-grid-item .grid-line {
+    top: 0;
+    background-color: black;
+    opacity: 0.15;
+    min-height: 100%;
+    width: 1px;
+    position: absolute;
+}
+
+.flame-chart-grid-item .timestamp {
+    top: 2px;
```

Comment:
this is fine but you could add the padding and width for the label here instead of doing it manually in code.

Reply:
I'll add left padding to css but i need to update width in code since it is dynamic

Reply:
sg. now that the padding is in the css, the width can be the full value without requiring you to do the math of computing the width without padding.

Reply:
I compute the width so that `text-align` end does what I need it to. Otherwise the div will be as wide as the text and it will be on the left side of the GridItem, not right next to the line on the right side.

Reply:
and since it's absolutely positioned, setting width to 100% just makes it the same width as it's parent but shifted over 4px, which makes it go out of the desired bounds.